src/main/kotlin/com/fantasy/ladbe/socket/service/BidLogSocketService.kt

```diff
@@ -0,0 +1,44 @@
+package com.fantasy.ladbe.socket.service
+
+import com.fantasy.ladbe.handler.exception.BusinessException
+import com.fantasy.ladbe.handler.exception.Exceptions.*
+import com.fantasy.ladbe.model.BidLog
+import com.fantasy.ladbe.repository.*
+import com.fantasy.ladbe.socket.dto.BidLogSocketDto
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class BidLogSocketService(
+    val bidLogRepository: BidLogRepository,
+    val playerRepository: PlayerRepository,
+    val bidderRepository: BidderRepository,
+    val auctionRepository: AuctionRepository
+) {
+
+    fun saveBidLog(
+        requestBidLog: BidLogSocketDto.Request.BidLogContent
+    ): BidLogSocketDto.Response.BidLogContent {
+
+        val auction = auctionRepository.findById(requestBidLog.auctionId)
+            .orElseThrow { throw BusinessException(AUCTION_NOT_FOUND) }
+        val bidder = bidderRepository.findById(requestBidLog.bidderId)
+            .orElseThrow { throw BusinessException(BIDDER_NOT_FOUND) }
+        val player = playerRepository.findById(requestBidLog.playerId)
+            .orElseThrow { throw BusinessException(PLAYER_NOT_FOUND) }
+
+        val bidLog = BidLog(
+            price = requestBidLog.price,
+            bidder = bidder,
+            auction = auction,
+            player = player
+        )
+        bidLogRepository.save(bidLog)
+
+        return BidLogSocketDto.Response.BidLogContent(
+            price = bidLog.price,
+            playerId = player.id,
+            bidderId = bidder.id,
```

Comment:
```suggestion
        return bidLogRepository.save(bidLog).toDto() 
```
다른 서비스 로직에서 작성하신 방식처럼, 이렇게 줄일 수 있을 것 같습니다. (특정 값이 빠지거나 추가되야 한다면 toDto() 메소드 내에서 할 수 있을 것 같아요.)

Reply:
넵!! 감사합니다.